song_translator/track/models.py

```diff
@@ -54,14 +54,14 @@ def __str__(self):
 
 
 class Track(models.Model):
-    name = models.CharField(max_length=64)
+    track_name = models.CharField(max_length=64)
```

Comment:
in fact `name` is fine, one can see that it belongs to the `track` from table name